GymBro/Views/RoutineViews/ShowSetsView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ShowSetsView.swift
+//  GymBro
+//
+//  Created by 陳宜葳 on 5/23/23.
+//
+//  This is the view for user to check what's already in the exercise list.
+
+import SwiftUI
+
+struct ShowSetsView: View {
+    
+    
+    // MARK: PROPERTY
+    
+    /// The exercise of each row. Should be an input from RoutineExerciseView.
+    var exercise: String
+    
+    /// The weight list has been added.
+    @State var weightList: [Int] = [120, 110, 110]
+    
+    /// The rep list has been added.
+    @State var repList: [Int] = [10, 15, 15]
+    
+    /// Show the drop down set list.
+    @State var showSet: Bool = false
+    
+    /// Show the EditSetView as a pop-up sheet
+    @State var showEditSheet: Bool = false
+    
+    
+    // MARK: BODY
+    
+    var body: some View {
+        
+        VStack {
+            Button {
+                showSet.toggle()
+            } label: {
+                HStack {
+                    Text(exercise)
+                        .foregroundColor(.black)
+                    Spacer()
+                    showSet ? Image(systemName: "chevron.down") : Image(systemName: "chevron.right")
+                }
+            }
+            
+            if showSet {
+                Divider()
+                VStack {
+                    ForEach(0..<3) { index in
+                        HStack {
```

Comment:
In the future, the HStack needs to add a `.onTapGesture` to edit the existing set information.